### PR TITLE
Gearman extension

### PR DESCRIPTION
**Pull request by hotrush**
---
name: 🎉 New Feature
about: Gearman extension
labels: enhancement

---

Related issue: #116 

### Description

This PR adds a possibility to isntall gearman extension

> In case this PR introduced TypeScript/JavaScript code changes:

- [x] I have written test cases for the changes in this pull request
- [x] I have run `npm run format` before the commit.
- [x] I have run `npm run lint` before the commit.
- [x] I have run `npm run release` before the commit.
- [x] `npm test` returns with no unit test errors.

### CODE REVIEW

**Comment by shivammathur** [comment]
@hotrush 
Please create a workflow to test this change.
If it is installing correctly, I will merge this.

File: src/scripts/gearman.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ini_file=$(php --ini | grep "Loaded Configuration" | sed -e "s|.*:s*||" | sed "s/ //g")
+sudo DEBIAN_FRONTEND=noninteractive apt-fast install -y php"$2"-dev
```
Comment (shivammathur) [inline src/scripts/gearman.sh]: `libgearman-dev` is required for this to compile.
```
sudo DEBIAN_FRONTEND=noninteractive apt-fast install -y php"$2"-dev libgearman-dev
```

**Comment by shivammathur** [comment]
Merged, will add the dependency and test.

**Comment by hotrush** [comment]
Hi, it was lightning fast :+1: 
Didn't know that sury's repo has gearman extension, and as i can see it uses 2.0.5 version that is quite outdated, it might not work with php7.3+

**Comment by shivammathur** [comment]
@hotrush 
Please request @oerdnj if he can provide `2.0.6`. Otherwise I will change the script to compile and install it.

**Comment by hotrush** [comment]
I think it might be better to wait for 2.0.7 with 7.4 support and  keep as is for now

**Pull request merged**